Sources/Nimble/Matchers/BeAKindOf.swift

```diff
@@ -1,14 +1,22 @@
 import Foundation
 
-#if _runtime(_ObjC)
```

Comment:
I'm using docker (you need to install this first: https://docs.docker.com/docker-for-mac/)

```bash
$ docker pull swiftdocker/swift:3.0.1
$ docker run -it -v `pwd`:/project swiftdocker/swift bash
$ cd project
$ swift test
```